opm/polymer/PolymerProperties.hpp

```diff
@@ -147,14 +147,14 @@ namespace Opm
             const auto& plyviscTable = tables->getPlyviscTables().getTable<PlyviscTable>(0);
 
 
-            c_vals_visc_ = plyviscTable.getPolymerConcentrationColumn();
-            visc_mult_vals_ =  plyviscTable.getViscosityMultiplierColumn();
+            c_vals_visc_ = plyviscTable.getPolymerConcentrationColumn().vectorCopy( );
```

Comment:
Why is `getNamedColumn().vectorCopy()` used some places and `exportColumn("NAME_STRING")` in others?